conda_forge_tick/auto_tick.xsh

```diff
@@ -15,7 +15,9 @@ from .git_utils import (get_repo, push_repo)
 # TODO: move this back to the bot file as soon as the source issue is sorted
 # https://travis-ci.org/regro/00-find-feedstocks/jobs/388387895#L1870
 from .migrators import *
-$MIGRATORS = [Version(), Compiler()]
+$MIGRATORS = [Version(), 
+# Compiler()
```

Comment:
Compiler migration is not complete though

Reply:
right, isn't that why it is commented out?